gcc/rust/hir/tree/rust-hir-item.h

```diff
@@ -60,8 +60,8 @@ class TypeParam : public GenericParam
 
   TypeParam (Analysis::NodeMapping mappings, Identifier type_representation,
 	     Location locus = Location (),
-	     std::vector<std::unique_ptr<TypeParamBound> > type_param_bounds
-	     = std::vector<std::unique_ptr<TypeParamBound> > (),
+	     std::vector<std::unique_ptr<TypeParamBound>> type_param_bounds
+	     = std::vector<std::unique_ptr<TypeParamBound>> (),
```

Comment:
Did your `clang-format` get updated to 11? :D

Reply:
```
$ clang-format --version
clang-format version 10.0.0-4ubuntu1
```

Not quite sure whats going on here to be honest lol.